src/main/scala/workflow/DelegatingTransformer.scala

```diff
@@ -0,0 +1,9 @@
+package workflow
+
+import org.apache.spark.rdd.RDD
+
+class DelegatingTransformer[T](override val label: String) extends TransformerNode[T] {
```

Comment:
This should probably be package protected. This basically just takes in a single fit dependency, and applies that fit dependency on the input data. It's what we currently chain estimator outputs to. I'll add some comments.